index.bs

```diff
@@ -1063,26 +1072,27 @@ Upon receiving an attestation statement, the [RP] shall:
     - If such extension doesn't exist, the attestation root certificate is dedicated to a single Authenticator model.
 
 3. If `alg` is ECDAA (e.g., "ED256", "ED512"):
-    - Look up the DAA root key from a trusted source.  The FIDO Metadata Service [[FIDOMetadataService]] provides an easy way to
-        access such information. The `claimedAAGUID` can be used for this lookup.
+    - Look up the DAA root key from a trusted source.  
+        For example, the FIDO Metadata Service [[FIDOMetadataService]] provides one way to access such information. 
+        The `claimedAAGUID` can be used for this lookup.
     - Perform DAA-Verify on `signature` for `rawData` (see [[!FIDOEcdaaAlgorithm]]).
     - Verify `rawData` syntax and that it doesn't contradict the signing algorithm specified in `alg`.
     - The DAA root key is dedicated to a single Authenticator model.
 
 4. Verify the contextual bindings (e.g., channel binding) from the clientData (see [[#authenticator-signature]]).
 
 5. Verify that user verification method and other authenticator characteristics related to this authenticator model, match the
-    [RP] policy. The FIDO Metadata Service [[FIDOMetadataService]] provides an easy way to access the authenticator
+    [RP] policy. For example, the FIDO Metadata Service [[FIDOMetadataService]] provides one way to obtain authenticator 
     characteristics.
 
 The [RP] MAY take any of the below actions when verification of an attestation statement fails:
 
 - Reject the request, such as a registration request, associated with the attestation statement.
 
 - Accept the request associated with the attestation statement but treat the attested Scoped Credential as one with surrogate
-    basic attestation (see [[#attestation-types]]), if policy allows it. If doing so, there is no cryptographic proof that the
-    Scoped Credential has been generated by a particular Authenticator model. See [[FIDOSecRef]] and [[UAFProtocol]] for more
-    details on the relevance on attestation.
+    basic attestation (see [[#sctn-attestation-types]]), if policy allows it. If doing so, there is no cryptographic proof 
+    that the Scoped Credential has been generated by a particular Authenticator model. See [[FIDOSecRef]] and [[UAFProtocol]]
+    for more details on the relevance on attestation.
```

Comment:
Not your fault, but probably should be "relevance to attestation" at the end there.